client/java-armeria/src/main/java/com/linecorp/centraldogma/client/armeria/ArmeriaCentralDogma.java

```diff
@@ -1092,8 +1093,17 @@ private static String toString(AggregatedHttpResponse res) {
         return unsafeCast(Entry.ofText(revision, entryPath, content0));
     }
 
-    private static <T> Entry<T> entryAsJson(Revision revision, JsonNode node, String entryPath) {
-        return unsafeCast(Entry.ofJson(revision, entryPath, getField(node, "content")));
+    private static <T> Entry<T> entryAsJson(Revision revision, JsonNode node, String entryPath,
+                                            EntryType type) {
+        final JsonNode content = getField(node, "content");
+        if (type == EntryType.JSON5) {
```

Comment:
This method isn't called with `EntryType.JSON5`?

I guess users want to retrieve the JSON5 entry as a `JsonNode` or text with comments. So it will be
```java
Entry<JsonNode> entry = ...;
JsonNode content = entry.content(); // 주석 제거됨
String contextText = entry.contentAsText(); // 주석 보임
```
So I guess we have to use `EntryType` for JSON5 with `JsonNode` even though the bytes of String are transported.

Reply:
@minwoox
For `Entry` of `EntryType.JSON5` to give `JsonNode` with `content()` method, it should be defined as `JSON5(JsonNode.class)` which will make it committed as PLAIN JSON.

So more viable option here would be..:
```java
Entry<String> entry = ...;
JsonNode contentAsJson = entry.contentAsJsonNode(); // without comments
String content = entry.content(); // with comments
```

And type parameter of `Entry<T>` should be in line with the one for `Query<T>` so I think we can give following entries for JSON5 files depending on the query type...:
* `Query<JsonNode>` - `Query.ofJson()` or `Query.ofJsonPath()` -> `Entry<JsonNode>`
* `Query<String>` - `Query.ofText()` -> `Entry<String>` - `Entry.ofJson5()`
* `Query<?>` - `Query.identity()` -> `Entry<String>` - `Entry.ofJson5()`

Hope to get some advice for this 😅..

Reply:
> JsonNode contentAsJson = entry.contentAsJsonNode(); // without comments

I'm reluctant to add this method to `Entry` because an `Entry` can be a text file.

> JSON5(JsonNode.class) which will make it committed as PLAIN JSON.

Can we specially treat it on the client/server side? I thought we can transport/commit in the form of String and just validate if it's really a JSON5 before transporting/commiting.

> Query<String> - Query.ofText() -> Entry<String> - Entry.ofJson5()

It's the same. Can we just do?:
`Query<JsonNode>` - `Query.ofJson5()` -> `Entry<JsonNode>` - `Entry.ofJson5()`

Reply:
> I'm reluctant to add this method to Entry because an Entry can be a text file
`Entry` already has this method since it implements `ContentHolder` (actually, the method name is `contentAsJson()`... 😂)

Reply:
So basically, 
- We transmit/store the JSON5 as a plain String.
- But before transmitting/storing, validate the String if it's really in the form of JSON5.
- Use `Query<JsonNode>` and get `Entry<JsonNode>`.
  - In the JSON5 `Entry`, it has the String that contains comments and we can retrieve it via `entry.contentAsText()`.
  - We can retrieve the `JsonNode` via `entry.content()` which cannot have comments.
  
I could probably miss something, so let me know your opinion. 😄

Reply:
> Entry already has this method since it implements ContentHolder (actually, the method name is contentAsJson()... 😂

Oops, I missed that. 😅

Reply:
> Can we specially treat it on the client/server-side? I thought we can transmit/commit in the form of String and just validate if it's really a JSON5 before transmitting/committing.

Okay, let me try this..!

Reply:
By the way, was there any reason that we wanted to introduce `EntryType.JSON5`?
While talking with other maintainers, we can probably retrieve the JSON5 entry without introducing new `EntryType.JSON5` 😅 :
```java
// Use the file extension to distinguish JSON/JSON5.
Entry<JsonNode> json5Entry = Entry.ofJson(..., "/foo.json5", ...);

// The file extension is also used for the query.
Query<JsonNode> json5Query = Query.ofJson("/foo.json5);
```

Reply:
@jrhee17 Thanks for the advice..! I am heading into the same direction.
But, here's one thing that I am not sure of.

Now that we need to send JSON5 string to the client, the shape of `EntryDto<T>` should also be changed.
Maybe we can also introduce `String contentAsText` property in there but it sounds like a bit redundant in most of cases.

Or we can just remove the type parameter `T` of `EntryDto` and just use `Object content` though we would lose type safety by doing so..

I am more inclined to the 2nd option since `EntryDto<?>` is being used all across the place.